tasks.c

```diff
@@ -378,6 +378,7 @@ PRIVILEGED_DATA static volatile BaseType_t xNumOfOverflows = ( BaseType_t ) 0;
 PRIVILEGED_DATA static UBaseType_t uxTaskNumber = ( UBaseType_t ) 0U;
 PRIVILEGED_DATA static volatile TickType_t xNextTaskUnblockTime = ( TickType_t ) 0U; /* Initialised to portMAX_DELAY before the scheduler starts. */
 PRIVILEGED_DATA static TaskHandle_t xIdleTaskHandle = NULL;                          /*< Holds the handle of the idle task.  The idle task is created automatically when the scheduler is started. */
+const UBaseType_t uxTopUsedPriority = configMAX_PRIORITIES - 1U;

```

Comment:
Can you please add a comment to say why it is here - otherwise it might be confusing to see a const variable that is apparently not used.  It can be like the comment added the variable is cast to void.

Reply:
Of course. Added in latest commit